trillian/examples/deployment/docker/ctfe/README.md

```diff
@@ -13,9 +13,19 @@ serving production logs!
   - github.com/google/certificate-transparency-go
 
 The instructions below assume you've checked out the repositories within
-`~/git/`, but if you have them in another location then just use a different
+`/workspaces/`, but if you have them in another location then just use a different
 path when you run the command.
 
+```bash
+# Terminal 1
+export GIT_HOME=/workspaces
```

Comment:
nit: Perhaps change these to use curly brackets like `${GIT_HOME}` to match other variable references in here?

Reply:
Done